Voxels

andy645jh opened the pull request:
Change manifest app-id

TingPing commented on org.flatpak.HelloVoxels.json:
```diff
@@ -0,0 +1,22 @@
+{
+    "app-id": "org.flatpak.HelloVoxels",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
```
Please move to `18.08`.

TingPing commented on org.flatpak.HelloVoxels.json:
```diff
@@ -0,0 +1,22 @@
+{
+    "app-id": "org.flatpak.HelloVoxels",
```
Needs an app-id controlled by upstream.

TingPing commented:
This isn't an actual app, just the template, forget to push something?

barthalion commented:
Git isn't the perfect tool to host binary files. Maybe try creating separate git repo and creating a release so a tarball gets generated?

andy645jh commented:
@TingPing yes, I forgot set some files up ,now  I already added the missing .xml files and updated the manifest file with the information about app and I took the advice from @barthalion and put the binary files into a server, now I have a question, should I do a new pull request? or is a push enough since this request is already open?

bilelmoussaoui commented:
The app-id seems to be wrong. You don't own that domain name... Plus is there a reason for using the GNOMD runtime instead of the Freedesktop one?
